examples/java/spring-boot/src/main/java/uk/gov/api/springboot/domain/model/repositories/ApiRepository.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.api.springboot.domain.model.repositories;
+
+import java.util.List;
+import uk.gov.api.springboot.domain.model.Api;
+
+public interface ApiRepository {
```

Comment:
Could this instead be a `Provider`?

Reply:
Thoughts @nahteb?

Reply:
I think it makes sense as a repository - not too sure what you mean by `provider` here though?

Reply:
So the `Provider` here is the actual Domain-specific thing, that provides the APIs that it's aware of.

Thinking about it, I would say this is more a `Storage`, as it's the underlying storage mechanism, and the `ApiService` could instead interact with a `Provider` object, instead of the raw `Storage`

Reply:
ah yep I get what you mean by `Provider` now. So are you thinking rename this as ApiStorage? And later introduce a Provider?

Reply:
Yeah I think that would be good, to keep it aligned with our domain model as we're currently thinking of it